lib/tasks/cortex/core/media.rake

```diff
@@ -29,15 +29,17 @@ namespace :cortex do
                            },
                          metadata:
                            {
-                             styles: {
-                               large: {geometry: '1800x1800>', format: :jpg},
-                               medium: {geometry: '800x800>', format: :jpg},
-                               default: {geometry: '300x300>', format: :jpg},
-                               mini: {geometry: '100x100>', format: :jpg},
-                               micro: {geometry: '50x50>', format: :jpg},
-                               post_tile: {geometry: '1140x', format: :jpg}
+                            image_styles: {
```

Comment:
I think you were asked to change it because using Paperclip's hook to prevent post processing would have been ideal, but we discovered that wasn't possible. I'm okay with deleting from the original `styles` key and producing a new hash for use in non-image assets.